linera-rpc/src/lib.rs

```diff
@@ -40,3 +40,8 @@ pub struct HandleCertificateRequest {
 }
 
 pub const FILE_DESCRIPTOR_SET: &[u8] = tonic::include_file_descriptor_set!("file_descriptor_set");
+
+#[cfg(not(target_arch = "wasm32"))]
+pub const CERT_PEM: &str = include_str!(concat!(env!("OUT_DIR"), "/self_signed_cert.pem"));
+#[cfg(not(target_arch = "wasm32"))]
+pub const KEY_PEM: &str = include_str!(concat!(env!("OUT_DIR"), "/private_key.pem"));
```

Comment:
Maybe this could go into `transport`, where we abstract over the other things that need to be conditionally compiled?

Reply:
I think no.
Because the keys would be in "grpc/transport.rs". However, the TLS is independent of GRPC, so it would not be adequate to move to that place.

Reply:
No, not yet.